File: Contagem.java
import java.util.Scanner;

public class Contagem {
    public static void main(String[] args) {
        Scanner prompt = new Scanner(System.in);
        int i = 0; // Inicialize i com 0
        int n, nota, cont = 0; // Inicialize cont com 0

        System.out.println("Quantidade de Alunos: ");
        n = prompt.nextInt();

        while (i < n) {
            i = i + 1;
            System.out.println("Nota do Aluno " + i + ":");
            nota = prompt.nextInt();

            if (nota >= 50) {
                cont = cont + 1; // Incrementa cont quando a nota é maior ou igual a 50
            }
        }

        System.out.println("São " + n + " alunos");
        System.out.println("São " + cont + " aprovados");
        prompt.close(); // Feche o Scanner quando não for mais necessário
    }
}

File: contagem.js
const readlineSync = require('readline-sync');

class Contagem {
    constructor() {
        this.n = 0;
        this.nota = 0;
        this.cont = 0;
        this.i = 0;
    }

    iniciarContagem() {
        console.log("Quantidade de Alunos: ");
        this.n = parseInt(readlineSync.question());

        while (this.i < this.n) {
            this.i = this.i + 1;
            console.log("Nota do Aluno: " + this.i);
            this.nota = parseInt(readlineSync.question());

            if (this.nota < 50) {
                this.cont = this.cont + 1;
            }
        }

        console.log("São " + this.n + " alunos");
        console.log("São " + this.cont + " aprovados");
    }
}

const contador = new Contagem();
contador.iniciarContagem();
